### PR TITLE
[CI:DOCS] Fix up example description of podman-export/export commands

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by flouthoc — APPROVED** [review]
LGTM

File: docs/source/markdown/podman-import.1.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Set variant of the imported image.
 
 ## EXAMPLES
 
+Import the selected tar ball into a new image, specifying the CMD, Entryping and labels:
```
Comment (vrothberg) [inline docs/source/markdown/podman-import.1.md:55]:
```suggestion
Import the selected tar ball into a new image, specifying the CMD, ENTRYPOINT and LABEL:
```

File: docs/source/markdown/podman-import.1.md [hunk 2]
```diff
@@ -64,6 +65,7 @@ Storing signatures
 db65d991f3bbf7f31ed1064db9a6ced7652e3f8166c4736aa9133dadd3c7acb3
 ```
 
+Import the selected tar ball into a new image, specifying the CMD, Entryping and labels:
```
Comment (vrothberg) [inline docs/source/markdown/podman-import.1.md:68]: Same here

File: docs/source/markdown/podman-import.1.md [hunk 3]
```diff
@@ -73,22 +75,14 @@ Writing manifest to image destination
...
-Getting image source signatures
-Copying blob e3b0c44298fc skipped: already exists
-Copying config ae9a27e249 done
-Writing manifest to image destination
-Storing signatures
-ae9a27e249f801aff11a4ba54a81751ea9fbc9db45a6df3f1bfd63fc2437bb9c
-```
-
 
+Import the a new tagged image from stdin in quiet mode:
```
Comment (vrothberg) [inline docs/source/markdown/podman-import.1.md:79]:
```suggestion
Import a new tagged image from stdin in quiet mode:
```

File: docs/source/markdown/podman-import.1.md [hunk 4]
```diff
@@ -101,6 +95,7 @@ Storing signatures
 db65d991f3bbf7f31ed1064db9a6ced7652e3f8166c4736aa9133dadd3c7acb3
 ```
 
+Import a named image from a tarball via a URL:
```
Comment (vrothberg) [inline docs/source/markdown/podman-import.1.md:98]: tarball vs tar ball (above)

**Pull request merged**